src/OpenColorIO/Config.cpp

```diff
@@ -3971,6 +3971,38 @@ ConstProcessorRcPtr Config::getProcessor(const ConstContextRcPtr & context,
     return getProcessor(context, dt, direction);
 }
 
+ConstProcessorRcPtr Config::getProcessor(const ConstNamedTransformRcPtr & namedTransform,
+                                         TransformDirection direction) const
+{
+    ConstContextRcPtr context = getCurrentContext();
+    return getProcessor(context, namedTransform, direction);
+}
+
+ConstProcessorRcPtr Config::getProcessor(const ConstContextRcPtr & context,
+                                         const ConstNamedTransformRcPtr & namedTransform,
+                                         TransformDirection direction) const
+{
+    // This gets or calculates the named transform in the requested direction. The result
+    // is always applied in the forward direction.
+    auto transform = NamedTransformImpl::GetTransform(namedTransform, direction);
+    return getProcessor(context, transform, TRANSFORM_DIR_FORWARD);
```

Comment:
This is perfect.  

However, it makes me realize now that my reply to your issue #1674 was wrong.  What you have here is what I was thinking of but I wrote "getTransform" instead of "GetTransform".  But now that I check, it seems that GetTransform was never added to the public API.  That was an oversight on our part because that was how I was intending people would feed the getProcessor call (since it removes the need for the caller to handle which transform directions are actually in the NamedTransform).

Sorry for missing that.  I understand better now why you felt adding these methods was important.  :)

If you have time, it would be nice to bring GetTransform forward to the public API.  (But no worries, we could do it later too.)

Reply:
Thanks Doug. I can try to bring `GetTransform` forward to the public API this evening.